proofOfConcept/ColorPickerAlpha/README.md

```diff
@@ -0,0 +1,20 @@
+<h1> Project Color Picker Alpha </h1>
+<p>by Samantha Difeo, Tyrone Wong and Mathuran Sadagopan
+</p>
+
+<img src="./Resources/images/colorPickerAlpha.png" />
+
+<h2>Overview</h2> 
+
+Have you ever struggled to sample the color of a picture or app on windows? Then project Color Picker Alpha is the solution for you! This app allows you to pick the color anywhere on your screen. By clicking anywhere on the screen, the color at that point is immediately copied to clipboard. When in RGB mode, the color is displayed and copied to clip board in RGB. By selecting the toggle button, the view and clipboard switches to HEX. Our team has also added the abiltiy to create a color history pallete by simply saving the colors you have selected. Then by selecting the color on the pallete your color is immediately copied back onto your clipboard.
```

Comment:
- capitalize "windows"
- "This app allows you to pick the color anywhere on your screen. By clicking anywhere on the screen" is repetitive. Maybe "This app allows you to pick the color anywhere on your monitors. Click once on the screen to have the color selected and automatically copied to your clipboard."
- clip board is one word, the first time it is written wrong
- ability is spelled wrong
- pallete is spelled palette
- Last sentence switch to something like "Afterwards, selecting the color on the palette brings the color back into focus and immediately copied back to your clipboard"

Reply:
- Maybe instead of "pick the color anywhere", "extract a color on any monitor"